Pod/Support/SharedSupport/FitNesseRoot/OCSlimProjectExamplePage/TestPage/content.txt

```diff
@@ -1,13 +1,32 @@
 !1 OCSlimProject Example Test Page
-
-Below is a simple Decision Table based Acceptance Test. It sends input to an instance of a Fixture which is implemented in your Xcode project. This is done by adding a class of the same name e.g. 'FixtureExample.swift' to your Acceptance Test target. 
+Below is a simple Decision Table based Acceptance Test. It sends input to an instance of a Fixture which is implemented in your Xcode project. This is done by adding a class of the same name e.g. 'FixtureExample.swift' to your Acceptance Test target.
```

Comment:
Great intro, comment to re-inforce information that is hard to grok if you don't really understand how it all works.